assets/js/src/main.js

```diff
@@ -4,3 +4,4 @@ import '../vendor/bling';
 import '../vendor/accessible-modal-dialog';
 import './sidebar';
 import './app';
+import '../vendor/svg4everybody.min';
```

Comment:
I am not super fan of loading the whole script just for IE, but there is no other option. Internet 10+ does not have conditional comments.